tests/pytests/test_http_monitor.py

```diff
@@ -0,0 +1,104 @@
+# =============================================================================
+# 2018+ Copyright (c) Artem Ikchurin <artem.ikchurin@gmail.com>
+# All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# =============================================================================
+
+import sys
+sys.path.insert(0, "")  # for running from cmake
+import pytest
+import zlib
+import json
+from datetime import datetime
+try:
+    import urllib.request as urllib_req
+except ImportError:
+    import urllib2 as urllib_req
+from statistics_checker import MonitorStatsChecker
+from statistics_checker import categories_combination
+
+import elliptics
+
+path_categories = {"/cache": elliptics.core.monitor_stat_categories.cache,
+                   "/io": elliptics.core.monitor_stat_categories.io,
+                   "/commands": elliptics.core.monitor_stat_categories.commands,
+                   "/backend": elliptics.core.monitor_stat_categories.backend,
+                   "/stats": elliptics.core.monitor_stat_categories.stats,
+                   "/procfs": elliptics.core.monitor_stat_categories.procfs,
+                   "/top": elliptics.core.monitor_stat_categories.top,
+                   "/all": elliptics.core.monitor_stat_categories.all}
+
+
+@pytest.fixture(scope="session")
+def address(servers):
+    address = {}
+    address['remote'], address['port'] = zip(servers.remotes, servers.monitors)[0]
```

Comment:
`zip` here is overused, you can just use `servers.remotes[0], servers.monitors[0]`. `zip` will "zip" whole lists.